apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -2369,6 +2370,54 @@ nameservice_transaction_revoke(MinerAddress, MinerPubkey) ->
     test_missing_address(account_id, Encoded, fun get_name_revoke/1),
     ok.
 
+nameservice_transaction_subname(MinerAddress, MinerPubkey) ->
```

Comment:
Should not this test be running only for `ct-lima` CI job (or equivalent from point of view of consensus) and later? Is it conditional somewhere and I am not noticing it?

Or at least what is being tested should be different depending on consensus version e.g. not accepted before Lima, accepted from Lima.